internal/ingress/controller/parser/parser.go

```diff
@@ -1120,6 +1120,11 @@ func overrideRouteProtocols(route *kong.Route, anns map[string]string) {
 }
 
 func overrideRouteHTTPSRedirectCode(route *kong.Route, anns map[string]string) {
+
+	if annotations.HasForceSSLRedirectAnnotation(anns) {
+		route.HTTPSRedirectStatusCode = kong.Int(302)
+	}
```

Comment:
You probably also want to set the protocols in route to be `https` (and/or `grpcs`).

Reply:
Checking it.